README.md

```diff
@@ -13,8 +13,8 @@ Link to the [live demo](http://sandbox.cmf.symfony.com/)
 
 ## Getting started
 
-You can run the sandbox on your system, or in a virtualbox VM using Vagrant. For the latter, see
-"Getting started using Vagrant"
+You can run the sandbox on your system if you have Apache or Nginx installed or
+[use Homestead](https://symfony.com/doc/current/setup/homestead).
```

Comment:
we should explain that one wants to use the parameters file for jackalope-doctrine-dbal . i think our tutorials still assume sqlite - should we use sqlite in homestead by default?